### DISCUSSION TITLE
Shelly dimmer 2 low dim value problem

### DISCUSSION THREAD

**Discussion by mtbhenkie** [arendst/Tasmota#17223] [Q&A]
Hi,
I am replacing my zwave modules with wifi-modules.
I have a problem with low dim value for a Shelly Dimmer 2.
I can't put the dimmer on with a dim value lower than about 40. Nothing happens.
I first have to set the dimmer on with value 40, then delay 80, and next dim to 4 (the lowest value of the used lamp).
As long as the lamp is on, I can change values higher and back to 4 and it keeps working good; however, after setting dim to 0, same problem again.
I tried to solve it with setting ShdWarmupBrightness to 20 (other values tried), but that does not help.
What can be the problem with the low dim value here? I think it will be very simple, but I can't find the solution.
I flashed Tasmota version 12.2.0, and STM32 version 51.6.
Additional, I can't get the parameter ShdWarmupBrightness after first setting not lower than 10 (lower values are giving response of last higher value).

**Comment by githubkusi** [comment]
I came to the same conclusion: you need to enable Shelly Dimmer 2 with a higher brightness value, then delay, then you can lower the value. If you start with a low value, the lamp stays dark. Using Homeassistant and stock Shelly firmware.

**Comment by Fabesson** [comment]
This would be a nice function. In the stock shelly firmware this function is also included as "Warm Up" setting.
